clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -139,6 +140,7 @@
         "@types/jest": "^24.0.18",
         "@types/node": "^12.6.2",
         "jest": "^24.9.0",
+        "jest-sonar-reporter": "^2.0.0",
```

Comment:
Did all needed legal checks was done for this dependency?

Reply:
It is used in other broadcom projects